server/controllers/general.js

```diff
@@ -0,0 +1,114 @@
+import nodemailer from 'nodemailer';
+import dotenv from 'dotenv';
+import jwt from 'jsonwebtoken';
+import models from '../models';
+
+dotenv.config();
+const jwtSecret = process.env.JWT_SECRET;
+
+const User = models.User;
+const Group = models.Group;
+
+export default {
+  // Load everyone registered on PostIt
+  getallusers: (req, res) => {
+    const offset = req.params.offset;
+    const limit = req.params.limit;
+
+    User.findAndCountAll({ attributes: ['id', 'firstName', 'lastName', 'email', 'phone'], order: [['firstName', 'ASC']], offset, limit })
+      .then((allUsers) => {
+        res.status(200).send(allUsers);
```

Comment:
Your json response does not give a real interpretation of what it is returning to whoever is consuming your api